### PR TITLE
Keras Quickstart

### PR DESCRIPTION
**Pull request by sjyothi54**
**Ticket URL:** https://issues.newrelic.com/browse/NR-157311
**Description:** Created Keras Quickstart
## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/mlops/keras/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+slug: keras
+description: |
+  ## Why monitor your Keras?
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:3]: Why monitor your Keras models?
Keras provides tools for building deep learning models with ease and efficiency. Monitoring Keras models during their training and deployment becomes crucial for various reasons:

Improve performance:
Training Keras models, especially large ones, can be computationally intensive. By monitoring metrics like training time, memory usage, and GPU utilization, you can identify bottlenecks and make necessary optimizations.

Model health:
It's vital to track the health of your models in terms of training loss, validation accuracy, and other metrics to ensure that they are learning correctly and not overfitting or underfitting.

Data quality:
Ensuring the quality and consistency of input data is paramount. Monitoring can help identify anomalies or inconsistencies in the data that could adversely affect model performance.

Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:3]: Why monitor your Keras models during inference?
Once Keras models are trained and deployed, monitoring during the inference phase becomes paramount for various reasons:

Improve performance:
Inference, especially with large models, can be computationally intensive. Monitoring metrics like inference latency, memory usage, and GPU utilization can identify bottlenecks, allowing for necessary optimizations.

Model health:
Over time, the distribution of input data might change, a phenomenon known as data drift. Monitoring allows for the early detection of such scenarios, ensuring the model remains robust and accurate.

Data quality:
As models make predictions on new, unseen data, ensuring the quality and consistency of input data is vital. Monitoring can help detect anomalies or inconsistencies in real-time data that could adversely affect model output.

File: quickstarts/mlops/keras/config.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  Keras is an open-source library that provides a Python interface for artificial neural networks. Monitoring Keras is essential for maintaining the smooth operation of the system and ensuring the delivery of precise data insights.
+  This necessity arises due to the following factors:
+  ### Improve performance:
+  Keras can be slow, especially for large models. By monitoring the Keras, you will be able to identify areas where performance can be improved.
+  ### Model health: 
+  Keep track of the model's health and reliability over time to maintain its accuracy and relevance.
+  ### Data quality: 
+  Monitor data preprocessing and input quality to maintain the integrity of model inputs.
+  
+  ## Comprehensive monitoring quickstart for Keras
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:13]: Comprehensive monitoring for Keras models during inference
Properly monitoring your Keras models during the inference phase ensures optimal performance and sustained accuracy. Tracking key metrics will help maintain model health and swiftly detect potential issues.

File: quickstarts/mlops/keras/config.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  Monitoring your Keras AI is an important step in ensuring the quality, accuracy, performance, compliance, user experience, and decision-making. By tracking key metrics, you will identify and address errors, improve performance, and comply with regulations. 
+
+  ## What’s included in the Keras quickstart?
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:16]: What’s included in the Keras monitoring quickstart?
The New Relic Keras monitoring quickstart offers specialized out-of-the-box reporting for inference:

Real-time visibility: Gain insights into the performance of your Keras models, tracking metrics like inference latency and throughput in real-time.
Data Integrity Checks: Ensure that the input data for inference aligns with expected formats and does not have any anomalies that could affect model outputs.
Proactive alerts: Receive immediate notifications about critical issues affecting your Keras model's performance or reliability during inference.
Resource Utilization: Monitor the computational resources (CPU, GPU, memory) being used during the inference process to optimize deployments and control costs.

File: quickstarts/mlops/keras/config.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  New Relic Keras monitoring quickstart provides quality out-of-the-box reporting:
+  - Real-time visibility: Get insights into the performance of your Keras in real time
+  - Security insights: Get insights into the security of your Keras and identify potential threats
+  - Be proactively alerted about critical issues related to your Keras's status
+  - Gain code-related insights about your Keras's health and status
+summary: |
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:22]: Boost the performance of your Keras models by integrating with New Relic, providing comprehensive monitoring and optimization for the entire inference process.

File: quickstarts/mlops/keras/config.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+level: 
+  - New Relic
+authors:
+  - New Relic
+  - Jyothi Surampudi
+title: Keras
+documentation:
+  - name: Keras integration documentation
+    description: |
+     Implement monitoring and instrumentation for your Keras, and ensure that your Keras's data is integrated into New Relic for effective performance analysis and insights.
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:34]: Implement monitoring and instrumentation for your Keras models, and ensure that your monitoring data is integrated into New Relic for effective performance analysis and insights.

File: quickstarts/mlops/keras/config.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    url: https://github.com/newrelic-experimental/ml-performance-monitoring
+dataSourceIds:
+  -  bring-your-own-data
+keywords:
+  -  keras
+  -  keras ai
+  -  large language model
```
Comment (ykriger-newrelic) [inline quickstarts/mlops/keras/config.yml:41]: Remove that

**Comment by mdumpati** [comment]
Closing this PR because merged into below PR
https://github.com/newrelic/newrelic-quickstarts/pull/2044
